products.php

```diff
@@ -0,0 +1,51 @@
+<?php
+require_once 'src/Models/CategoryModel.php';
+require_once 'src/Services/ProductsDisplayService.php';
+require_once 'src/Models/ProductModel.php';
+require_once 'src/Factory/furnitureDatabaseConnector.php';
+require_once 'src/Entities/ProductsEntity.php';
+
+$db = furnitureDatabaseConnector::connect();
+define('NUM_OF_CATEGORIES', 11);
```

Comment:
How do you know how many categories there are? You can't just count how many there are in the database - in the real world this is changeable. 

Do you even need to know how many categories there are?